framework/src/task_manager/taskmgr_interface.c

```diff
@@ -451,6 +451,18 @@ int task_manager_set_handler(void (*func)(int signo, tm_msg_t *data))
 	return OK;
 }
 
+int task_manager_set_res_handler(void (*func)(int signo, tm_msg_t *data))
```

Comment:
It is difficult to distinguish set_handler and set_res_handler.
Please rename set_handler to know the purpose of function.